From eb613b7227f5c490f6bd5e0d55798521a76d7a6f Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Thu, 18 Oct 2018 10:15:37 +0800
Subject: [PATCH] arm64: dts: rockchip: set usb3 suspend_clk to 24MHz for
 rk1808

The default clock frequency of usb3 suspend_clk is
32KHz, this patch sets the clock frequency to 24MHz
which from the xin24m parent.

Change-Id: Ia516a0d7b6c69b87a1ad6c69c421504477e18742
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 370c39d3bb25..4dc5cf36bbef 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -141,6 +141,8 @@
 			 <&cru SCLK_USB3_OTG0_SUSPEND>;
 		clock-names = "ref_clk", "bus_clk",
 			      "suspend_clk";
+		assigned-clocks = <&cru SCLK_USB3_OTG0_SUSPEND>;
+		assigned-clock-rates = <24000000>;
 		#address-cells = <2>;
 		#size-cells = <2>;
 		ranges;
-- 
2.35.3

